packages/shared/src/lib/rank.ts

```diff
@@ -1,22 +1,3 @@
-export const NO_RANK = 'No rank';
```

Comment:
We should completely delete this file once the relevant hooks are deleted as well.

Reply:
Yup yup working on it :D

Reply:
Very soon 😅  I still forgot to remove some other stuff u mentioned below